webapp/channels/src/components/channel_header_menu/channel_header_menu.tsx

```diff
@@ -86,7 +86,9 @@ export default function ChannelHeaderMenu({dmUser, gmMembers, isMobile, archived
         channelTitle = <ChannelHeaderTitleGroup gmMembers={gmMembers}/>;
     }
 
-    const pluginItems = pluginMenuItems.map((item) => {
+    const isSharedChannel = channel?.shared || false;
```

Comment:
Why include the `|| false` in most of these? It doesn't seem necessary when it's immediately checked as a condition